features/web-nfc.yml

```diff
@@ -0,0 +1,27 @@
+name: Web NFC
+description: The `NDEFReader` API reads and writes messages to near-field communication (NFC) tags.
+spec: https://w3c.github.io/web-nfc/
+caniuse: webnfc
+compat_features:
+  - api.NDEFMessage
+  - api.NDEFMessage.NDEFMessage
+  - api.NDEFMessage.secure_context_required
+  - api.NDEFMessage.records
+  - api.NDEFReader
+  - api.NDEFReader.NDEFReader
+  - api.NDEFReader.secure_context_required
+  - api.NDEFReader.makeReadOnly
```

Comment:
https://chromestatus.com/feature/5700853265596416 claims this shipped in Chrome 100. When we added this in BCD in https://github.com/mdn/browser-compat-data/pull/18279 we were already past the release of Chrome 100, so I suspect this is an error in BCD. Can you comment it out here as a later addition?

Reply:
I've updated BCD in https://github.com/mdn/browser-compat-data/pull/23104.